data_steward/cleaners/clean_cdr.py

```diff
@@ -97,33 +29,34 @@ def clean_ehr_dataset(project=None, dataset=None):
         LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
 
     LOGGER.info("Cleaning ehr_dataset")
-    _clean_dataset(project, dataset, ehr_consts.SQL_QUERIES)
+    rule_4.run_clean_rule_4(project, dataset)
+
 
 def clean_unioned_ehr_dataset(project=None, dataset=None):
     if dataset is None or dataset == '' or dataset.isspace():
         dataset = bq_utils.get_unioned_dataset_id()
         LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
 
     LOGGER.info("Cleaning unioned_dataset")
-    _clean_dataset(project, dataset, unioned_consts.SQL_QUERIES)
+    rule_4.run_clean_rule_4(project, dataset)
 
 
 def clean_ehr_rdr_dataset(project=None, dataset=None):
     if dataset is None or dataset == '' or dataset.isspace():
         dataset = bq_utils.get_ehr_rdr_dataset_id()
-        LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
+        clean_engine.LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
 
     LOGGER.info("Cleaning ehr_rdr_dataset")
-    _clean_dataset(project, dataset, combined_consts.SQL_QUERIES)
+    clean_engine.clean_dataset(project, dataset)
 
 
 def clean_ehr_rdr_unidentified_dataset(project=None, dataset=None):
     if dataset is None or dataset == '' or dataset.isspace():
         dataset = bq_utils.get_combined_deid_dataset_id()
-        LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
+        clean_engine.LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
```

Comment:
remove this and just use `LOGGER.info`.  It's already set up.  You don't have to funnel it through `clean_engine.LOGGER`.